internal/db/schema/migrations/oss/postgres/42/05_key_rewrap.up.sql

```diff
@@ -0,0 +1,26 @@
+
+-- make the required schema changes to upgrade the dependency:
```

Comment:
* This migration should be in a new migration directory since directory 42 has already been released. 
* The number in the filename needs to be changed if the new migration directory already has an '05' migration file

Reply:
We accidentally missed this when rebasing, but we're going to make this change in a separate PR instead since it should be fixed in `main`, not just on this branch.

Reply:
Why would this change already be on main?

Reply:
Sorry, I mean that it _should_ be fixed in `main`, not that it is already fixed on `main`

Reply:
https://github.com/hashicorp/boundary/pull/2371 will track the merging, it's just blocked right now